kratos/geometries/quadrature_point_geometry.h

```diff
@@ -280,7 +280,7 @@ class QuadraturePointGeometry
         p_geometry->SetData(rGeometry.GetData());
         return p_geometry;
     }
-    
```

Comment:
nice

Reply:
Thanks, was thinking about this quite some time.